scripts/update-android-docs.sh

```diff
@@ -72,7 +72,7 @@ function generate_docs() {
 
 function prepare_branch_with_documentation() {
   INTERIM_BRANCH_WITH_DOCUMENTATION="${BRANCH_WITH_DOCUMENTATION}_${1}"
-  git checkout $BRANCH_WITH_DOCUMENTATION
+  git checkout -f $BRANCH_WITH_DOCUMENTATION
```

Comment:
will this overwrite the changes to the build.gradle? can we change the build.gradle after the branch is changed?

Reply:
No. build.gradle will keep the same

Reply:
👍